CommandLine/emake-tests/Parsing/parser-test.cpp

```diff
@@ -114,7 +114,7 @@ TEST(ParserTest, Basics_NoSemicolon) {
 }
 
 TEST(ParserTest, SizeofExpression) {
-  ParserTester test{"sizeof 5"};  // we need to support sizeof (5)
+  ParserTester test = ParserTester::CreateWithCpp("sizeof 5");  // we need to support sizeof (5)
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax